nan.h

```diff
@@ -945,6 +948,56 @@ class Utf8String {
   char str_st_[1024];
 };
 
+# ifdef __MVS__
+class EbcdicString {
+ public:
+  inline explicit EbcdicString(v8::Local<v8::Value> from) :
+      length_(0), str_(str_st_) {
+    HandleScope scope;
+    if (!from.IsEmpty()) {
+      v8::Local<v8::String> string = from->ToString();
+      if (!string.IsEmpty()) {
+        size_t len = string->Length() + 1;
+        assert(len <= INT_MAX);
+        if (len > sizeof (str_st_)) {
+          str_ = static_cast<char*>(malloc(len));
+          assert(str_ != 0);
+        }
+        const int flags = v8::String::NO_NULL_TERMINATION;
+        length_ = string->WriteOneByte(reinterpret_cast<unsigned char*>(str_),
```

Comment:
`static_cast`

Reply:
I will change the underlying storage type to a an unsigned char* (because ebcdic is based on unsigned char)
But I will then have to to move the reinterpret_cast to the cast operators.

 inline char* operator*() { return reinterpret_cast<char*>(str_); }